README.md

```diff
@@ -44,3 +48,11 @@ Follow the steps to install the node packages required for both building and run
 4. By default `health/ping` call on `localhost:8080` gives a sample response
 
```

Comment:
We say "Build and Test" and we end with only the health check.
Maybe say it explicitly that we are only "testing" the successful compilation and build and running the service.
Not actually "Testing (unit or functional)" the service?